### ISSUE TITLE
Mobile Friendly section should be under Accessibility Category

### ISSUE THREAD

**Issue by rcanu** [GoogleChrome/lighthouse#9837]
**Summary**

I think the **Mobile Friendly section** should be placed under **Accessibility**. Or is there a reason why it is under **SEO section**? Or is it because SEO also involves having the site be mobile friendly?

![image](https://user-images.githubusercontent.com/3313421/66750801-ff3f1800-eebf-11e9-99cb-588a8da3df9e.png)

**Comment by patrickhulce** [comment]
You are correct that it is under SEO because mobile-friendliness is an important ranking consideration and these checks mirror the official "mobile friendly test" (https://search.google.com/test/mobile-friendly) from Google.

It's an interesting idea to mirror these audits under accessibility though, I like it! Accessibility means the site is accessible to everyone and these all apply to the usability of the site to a broad set of users :)

**Comment by midzer** [comment]
We could place it in both categories just as "Image elements have [alt] attributes"

**Comment by patrickhulce** [comment]
Yes @midzer that is the only proposal on the table right now, we certainly won't be removing these from SEO :)

**Comment by midzer** [comment]
Oooops, I've misread your "mirror" with "move". Sorry :D
